docs/index.rst

```diff
@@ -30,6 +30,11 @@ Document Structure
    Data Retrieval <start/getdata.rst>
    Custom Model Integration <start/integration.rst>
 
+.. toctree::
```

Comment:
Do you think it will be more reasonable to put it before `REFERENCE`: and after `OTHER COMPONENTS/FEATURES/TOPICS:`?

Reply:
Done